components/resume/company/company.tsx

```diff
@@ -1,33 +0,0 @@
-import React, { FC } from 'react';
-import Position from './positions';
-import Skills from './skills';
-
-interface Position {
-    title: string;
-    startDate: string;
-    endDate?: string;
-}
-
-interface CompanyProps {
-    name: string;
-    positions: Position[];
-    startDate: string;
-    endDate?: string;
-    skills: string[];
-    highlights: string[]
-}
-
-const Company: FC<CompanyProps> = ({ name, skills, positions, highlights }) => {
```

Comment:
This was just renamed to `Job`, guess too many other minor changes for git to flag this as a file rename and refactor.